code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -335,6 +335,16 @@
 	if(..())
 		playsound(user.loc, 'sound/emotes/dwoop.ogg', 50)
 
+/datum/emote/living/carbon/human/robot_tongue/slowclap
+	key = "slowclap"
+	key_third_person = "activates their slow clap processor."
+	message = "activates their slow clap processor."
```

Comment:
```suggestion
	key_third_person = "plays a slow clapping sound."
```

Reply:
https://youtu.be/wcnpZYmNAuI?si=YJVf7qHxVUBysXR6

Reply:
I just updated my original review with a more extensive opinion while you were responding if you want to read it. I did not remember that Portal phrase, but even though its a funny nod to a beloved franchise I still think it does not translate well into the game

Reply:
I guess I can agree about why a borg would have a slow clap processor in the first place, but if you used that logic elsewhere half the game would be removed + this isn't a HRP server. As for the wording of the emote, I disagree. I think how it is now is funnier and the only problem I see with it because of it's length it doesn't fully display in runechat (you can see this in the testing evidence). Thanks for the feedback though!